components/lib/inputnumber/InputNumber.js

```diff
@@ -448,27 +457,31 @@ export const InputNumber = React.memo(
                 default:
                     break;
             }
-
-            if (props.onKeyDown) {
-                props.onKeyDown(event);
-            }
         };
 
-        const onInputKeyPress = (event) => {
+        const onInputKeyUp = (event) => {
             if (props.disabled || props.readOnly) {
                 return;
             }
 
-            const code = event.which || event.keyCode;
+            if (props.onKeyUp) {
+                props.onKeyUp(event);
+
+                // do not continue if the user defined event wants to prevent
+                if (event.defaultPrevented) {
+                    return;
+                }
+            }
+
+            const key = event.key;
 
-            if (code !== 13) {
+            if (key !== 'Enter') {
                 // to submit a form
                 event.preventDefault();
             }
 
-            const char = String.fromCharCode(code);
```

Comment:
This definition is missing and is the cause of the error.

Reply:
Fixed!